hogben/optimise.py

```diff
@@ -263,7 +264,8 @@ def _contrasts_func(self,
             ]
 
             # Add to the initial Fisher information matrix.
-            g += self.sample.contrast_info(angle_times, [x[i]])
+            g += self.sample.contrast_info(angle_times,
```

Comment:
The reason I did not do that is because `fisher` in the other method is a `Fisher` object, while `g` here is the matrix itself (which is just a numpy array). But I am not too strongly opposed to changing it, but the idea was to use a different name for different things.

Reply:
OK, as discussed, lets add a comment saying what g is, and that will be fine